src/pages/registrationform.tsx

```diff
@@ -15,24 +14,15 @@ export const getServerSideProps = (async (context) => {
     context.res
   );
   if (loginProps?.isAuthorized) {
-    const supabase = getSupabase(loginProps.userID);
-    const { data } = await supabase
-      .from("users")
-      .select()
-      .eq("passage_user_id", loginProps.userID);
-      
```

Comment:
In case an existing user loads this page (by accident or otherwise), what do you think about checking for an existing user, and redirecting to the dashboard if an existing user is found?

Reply:
I hadn't thought about that, That seems like a good idea